<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="_8_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="nanugi_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="_pm_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="equal_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="_0_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="sqr_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>SQR</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="dot_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ce_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="_5_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="_7_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="_6_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="_gobhagi_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="_minus_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="_2_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="_plus_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="erase_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Erase</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clear_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="fraction_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="_4_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="_9_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="_1_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="sqrt_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>SQRT</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="_3_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="namuji_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="num_Edt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>590</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_7_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_8_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_9_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_4_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_5_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_6_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_1_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_2_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_3_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_0_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_plus_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>operator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_minus_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>operator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_gobhagi_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>operator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nanugi_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>operator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sqrt_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sqr_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fraction_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_pm_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>change()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ce_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear_one()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>erase_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>erase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>equal_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>result()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>546</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>namuji_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>operator()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dot_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_number()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_number()</slot>
  <slot>clear()</slot>
  <slot>operator()</slot>
  <slot>change()</slot>
  <slot>clear_one()</slot>
  <slot>erase()</slot>
  <slot>result()</slot>
 </slots>
</ui>
